lib/server.go

```diff
@@ -987,6 +987,14 @@ func (srv *Server) _handleHeaderBundle(pp *Peer, msg *MsgDeSoHeaderBundle) {
 		// if we're in the process of syncing.
 		_, isOrphan, err := srv.blockchain.ProcessHeader(headerReceived, headerHash, !srv.blockchain.isSyncing())
 
+		numLogHeaders := 2000
```

Comment:
This change doesn't do anything other than add logging, and the logging is very useful so I was lazy and left it here.